src/containers/PageContainer/elements/Header/elements/InfoButton/elements/InfoPanel/InfoPanel.tsx

```diff
@@ -1,47 +1,45 @@
-import { LINKS } from 'constants/nav';
+//- React Imports
 import React from 'react';
-import { Link } from 'react-router-dom';
+
+//- Components Imports
 import { ConnectWalletButton } from '../../../../elements';
 import { InfoButtonProps } from '../../InfoButton';
+
+//- Library Imports
+import classnames from 'classnames';
+
 import {
 	WILDER_WORLD_OPTIONS,
 	ZERO_TECH_OPTIONS,
 	ZNS_OTHER_OPTIONS,
 } from './InfoPanel.constants';
+
+//- Styles Imports
 import './_info-panel.scss';
 
 interface InfoPanelProps extends InfoButtonProps {}
 
 export const InfoPanel: React.FC<InfoPanelProps> = ({
 	isDesktop,
+	isWalletConnected,
 	onConnectWallet,
 }) => {
 	return (
 		<div className="info-panel__content">
-			<div className="info-panel__content-section connect-wallet">
-				<ConnectWalletButton
-					onConnectWallet={onConnectWallet}
-					isDesktop={isDesktop}
-				/>
+			<div
+				className={classnames('info-panel__content-section connect-wallet', {
+					'info-panel__content-section connect-wallet--is-wallet-connected':
+						!isWalletConnected,
+				})}
+			>
+				{isWalletConnected && (
+					<ConnectWalletButton
+						onConnectWallet={onConnectWallet}
+						isDesktop={isDesktop}
+					/>
+				)}
 			</div>
 			<div className="info-panel__content-section">
-				<div className="info-panel_nav info-panel__content-section">
```

Comment:
I removed this as it doesn't appear to be used? Please let me know if this is incorrect...Removing it also removed console errors.

Reply:
@domw30 yeah correct - that was my temporary solution for when we didn't have the mobile nav design yet (nav links in the dropdown). All good to remove!